app/destination/page.js

```diff
@@ -1,63 +1,57 @@
 "use client";
 
-import { useState } from 'react';
+import { useState } from "react";
 
-import styles from '@/components/destination/destination.module.css';
-import { AddWishlistItem } from '@/components/destination/AddWishlistItem';
+import { AddWishlistItem } from "@/components/destination/AddWishlistItem";
+import PlanetCard from "./planetCard";
 
-// TASK - React 1 week 2
-// Move this to its own file
-const PlanetWishlistItem = ({
-  name,
-  onRemove,
-  thumbnail,
-}) => {
-  return (
-    <div className={styles.wishlistItem}>
-      <img className={styles.wishlistItemThumbnail} src={thumbnail} alt="" />
-      <b>{name.toUpperCase()}</b>
-      <button onClick={onRemove}>remove</button>
-    </div>
-  );
-}
+export const Destinations = () => {
+	const [selectedPlanets, setSelectedPlanets] = useState([]);
+	const [planetCounter, setPlanetCounter] = useState(0);
```

Comment:
There's no need to create a state here! You can just calculate the length dynamically from the selectedPlanets like this:

const planetCounter = selectedPlanets.length